Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -26,15 +26,14 @@ public enum Team
     [HideInInspector]
     public Rigidbody agentRb;
     SoccerSettings m_SoccerSettings;
-    Renderer m_AgentRenderer;
```

Comment:
@andrewcoh A few unused variables from soccer. Just want to make sure there's no plans to bring these back.

Reply:
Ah I missed this. Good to remove.